src/material-experimental/mdc-snack-bar/_snack-bar-theme.scss

```diff
@@ -28,6 +28,12 @@
     @include mdc-snackbar-core-styles($query: $mat-theme-styles-query);
   }
 
+  // Usually this color is provided by the `mdc-snackbar-core-styles` mixin, but our styles
+  // to handle unthemed buttons have a higher specificity so we have to re-provide it here.
+  .mat-mdc-snack-bar-container .mat-mdc-snack-bar-action.mdc-snackbar__action {
```

Comment:
I don't think this will color the button as accent, like our legacy snack bar has. Should you merge in my change from https://github.com/angular/components/pull/21384/files to make sure the color is set to accent in light theme, and text in dark?

Reply:
No, it won't. I tried to match the pink-ish color that we get from MDC and which was in master when I made these changes.

![Angular_Material_-_Google_Chrome_2021-01-09_17-05-19](https://user-images.githubusercontent.com/4450522/104094967-dd883880-529c-11eb-8460-37731e05c983.png)